conda/cli/main_config.py

```diff
@@ -365,6 +365,29 @@ def execute_config(args, parser):
                                 (item, key))
         rc_config[key] = [i for i in rc_config[key] if i != item]
 
+    # Clear
+    for key, in args.clear:
```

Comment:
```suggestion
    for key in args.clear:
```

I don't think we need that comma, do we?

Reply:
ah, without the `,` notation, this iterates over an array object, e.g. key == [keyvalue], the comma notation pulls the element out of the array. This approach mirrors the iteration logic that `remove key` uses:
https://github.com/conda/conda/blob/448fc905536ffd5a39781aeb50076c7831e302dd/conda/cli/main_config.py#L391